src/components/navigation/navigation-item/navigation-item.component.html

```diff
@@ -0,0 +1,7 @@
+<a *ngIf="link" (click)="toggle(); $event.stopPropagation()" [class.has-arrow]="this.children.length > 0" [class.no-arrow]="this.indentWithoutArrow()" [routerLink]="link">

```

Comment:
Also, remove the use of `this` in the view, it is not needed as the component class is the context used for rendering the view

Reply:
Removed unnecessary 'this' in template.